src/core/Akka.Remote.Tests.Performance/ThreadPoolDispatcherRemoteMessagingThroughputSpec.cs

```diff
@@ -0,0 +1,23 @@
+using Akka.Configuration;
+using Akka.Remote.Tests.Performance.Transports;
+
+namespace Akka.Remote.Tests.Performance
+{
+    public class ThreadPoolDispatcherRemoteMessagingThroughputSpec : TestTransportRemoteMessagingThroughputSpec
+    {
+        public static Config ThreadPoolDispatcherConfig => ConfigurationFactory.ParseString(@"
+            akka.remote.default-remote-dispatcher {
+              type = Dispatcher
```

Comment:
Might need someone else to verify this, but this is the correct way to specify the `ThreadPoolDispatcher` in HOCON, no?